### ISSUE TITLE
Add to "Download Caddy"

### ISSUE THREAD

**Issue by zorbaTheRainy** [tailscale/caddy-tailscale#80]
It would be great if I could download this (with other plug-ins) from [Download Caddy](https://caddyserver.com/download).

I am aware I could use `xcaddy build` but I prefer to have Caddy do the building.  It is convenient.  

Thanks

**Comment by willnorris** [comment]
I had tried to register it once before and got errors.  Apparently it's still erroring:

![Image](https://github.com/user-attachments/assets/f6cb6ee4-4302-4212-833f-063fe98e6a9c)

Error ID, so you don't have to retype it: 5efc5f78-bbe4-4c4c-808f-13ac1112913d

@mholt, I think we might have talked about this once before (I can't remember)?  Is that something you can help with?

**Comment by willnorris** [comment]
> ^ Is that the version you're trying to claim? i.e. is this the latest tagged version?

That is the latest commit, yes.  We actually don't have any tagged version right now, though.  Would that be a problem?

**Comment by mholt** [comment]
Hmm no, it shouldn't. Is that line a Unary Expression? I am mobile so can't easily look it up right now but I don't think so... I wonder if our registration server is looking at an old version of the code or something?

**Comment by willnorris** [comment]
No, it's just a composite literal.  I'll try tagging a release just in case that is in fact causing problems.  Should probably do that anyway :)

**Comment by mholt** [comment]
It should totally be working. I'll try to investigate more when I have a chance. (Sorry!)
